rust/libnewsboat/src/utils.rs

```diff
@@ -265,8 +267,78 @@ pub fn unescape_url(rs_str: String) -> Option<String> {
     Some(result.unwrap().replace("\0",""))
 }
 
+pub fn get_command_output(cmd: &str) -> String {
+    let cmd = Command::new("sh").arg("-c").arg(cmd).output();
+    // from_utf8_lossy will convert any bad bytes to U+FFFD
+    cmd.map(|cmd| String::from_utf8_lossy(&cmd.stdout).into_owned())
+        .unwrap_or_else(|_| String::from(""))
+}
+
+pub fn run_command(cmd: &str, param: &str) {
+    let child = Command::new(cmd)
+        .arg(param)
+        .stdin(Stdio::null())
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
+        .spawn();
+    if let Err(error) = child {
+        log!(Level::Debug,
+             &format!("utils::run_command: spawning a child for \"{}\" failed: {}", cmd, error));
+    }
+
+    /* We deliberately *don't* wait for the child to finish. */
```

Comment:
Nitpick: Why not a `//` comment?

Reply:
I actually have no idea. Just something I typed… [Rust Style Guide](https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/guide.md) says to use `//`, so I'll change the code.